Cargo.toml

```diff
@@ -10,6 +11,7 @@ members = [
 ]
 default-members = [
     "rust/aes-gcm-siv",
+    "rust/crypto",
```

Comment:
Long-term, should `aes-gcm-siv` move into `crypto`? Or is it sufficiently different?

Reply:
Alternately, should `crypto` have a more specific name, even if it's exposed generically in Java for backwards-compatibility?

Reply:
I've moved `aes-gcm-siv` code into `rust/crypto`